pep-9999.rst

```diff
@@ -285,6 +285,21 @@ Two kinds of structured comments are accepted:
 * A ``# type: ignore`` comment at the end of any line, which suppresses all type
   errors in that line.
 
+``__all__``
+-----------
+
+Stubs support customizing star import semantics by defining a module-level
+variable called ``__all__``. ``__all__`` should be a list of strings. By
```

Comment:
At runtime it can be any iterable I believe, and it's not uncommon for it to be a tuple. Also, at runtime people sometimes build up `__all__` dynamically.

Perhaps we should say explicitly that the semantics are similar to using `__all__` at runtime, but a bit stricter.

Reply:
I think we can limit it to "iterable literals" in stubs, though (list, tuple, set, dict?), since I don't think building the list dynamically in stubs is or should be supported.

Reply:
I changed the wording to "list, set, or tuple literal" - what do you think?

Reply:
I think that's fine. But should we also add a style guide section recommending one particular form? I'm leaning towards no, it doesn't seem very important.

Reply:
We could suggest lists, but it's not that important.

Reply:
I added a short style guide section on ``__all__``.
I also tried to merge in the current type-stub-pep branch, which seems to have screwed up the diff, but I'm not sure how to fix it =S

Reply:
Maybe rebase the branch on srittau/master and force-push?

Reply:
That did the trick, thanks!